A few additions to the README.md (#1144)
MT-CTF/capturetheflag: mods/ctf/ctf_map/README.md

1. Select the `Capture the Flag` subgame.
2. Create a new world. You can select any mapgen you like.
3. Enable creative mode. This will enable `mapedit` mode.

> 5. *Optional:* Enable `worldedit`

Reviewer: ```suggestion
4. *Optional:* Enable `worldedit`
```

1. Select the `Capture the Flag` subgame.
2. Create a new world. You can select any mapgen you like.
3. Enable creative mode. This will enable `mapedit` mode.
5. *Optional:* Enable `worldedit`

### 3. Select an area

1. Decide where you will build your map. The area can be maximum 230x230 blocks in surface area, but it can be lesser.
2. Grant yourself the `ctf_map_editor` privilege by running `/grantme ctf_map_editor` or by running `/grantme all` which grants you all the privileges that will be useful for map making.
3. Run `/ctf_map editor`
4. Select "Create New Map"
5. Select a position to be one corner of the map.
6. Select a position in the opposite corner. Place it higher/lower than the first one to give the map height.
7. **Note:** you can change your area at any time by typing `/ctf_map editor` and pressing the "Corners" button.

> > **Important:** If you ever want to exit the world in which you are making the map and want to come back to it later, simply exit it (**do not** press "Finish Editing in the CTF map editor). When you want to start working on the map again, you can simply rejoin the world and continue. *Only when you have finished making the map should you run* `/ctf_map editor` . 

Author: Looks like there's a loophole. This means that `/ctf_map editor should only be run at the end. I'll have to check in with some of the map managers to verify this.

> > **Important:** If you ever want to exit the world in which you are making the map and want to come back to it later, simply exit it (**do not** press "Finish Editing in the CTF map editor). When you want to start working on the map again, you can simply rejoin the world and continue. *Only when you have finished making the map should you run* `/ctf_map editor` . 

Author: I had mistakenly resolved it

> > **Important:** If you ever want to exit the world in which you are making the map and want to come back to it later, simply exit it (**do not** press "Finish Editing in the CTF map editor). When you want to start working on the map again, you can simply rejoin the world and continue. *Only when you have finished making the map should you run* `/ctf_map editor` . 

Author: Hmm. Looks like I have to fix the wrong commit when I reach home.

- Many blocks have an indestructible variant
- Don't forget to add
	- Team chests
	- Indestructible blocks under the flag. The minimum is 5x5 blocks in surface area.
	- "Indestructible Barrier Glass" (`ctf_map:ind_glass`) around the edge of the world.
	- "Indestructible Red Barrier Glass" (`ctf_map:ind_glass_red`) for the build-time wall. This will disappear once the match begins.
	- "Indestructible Red Barrier Stone" (`ctf_map:ind_stone_red`) for underground build-time wall. This will turn to stone(`default:stone`) once the match begins.

### 5. Fill out information about the map

Run `/ctf_map editor`.
An explanation of some of the fields is given below.

#### Selecting the Map Area <sup><em>(*required)</em></sup>
1. Select a position to be one corner of the map.
2. Select a position in the opposite corner. Place it higher/lower than the first one to give the map height.

> **Note:** Run `/ctf_map editor` to fill out the rest of the fields.

> #### Map Enabled <sup><em>(*required)</em></sup>

Reviewer 2: ```suggestion
#### Map Enabled *(required)*
```

Author: Ig HTML has to go…

- Many blocks have an indestructible variant
- Don't forget to add
	- Team chests
	- The blocks under the flag should be indestructible. The minimum is 5x5 blocks in surface area.
	- "Indestructible Barrier Glass" (`ctf_map:ind_glass`) around the edge of the world.

> 	- "Indestructible Red Barrier Glass" (`ctf_map:ind_glass_red`) for the build-time wall. The setting to make it disappear upon the beginning of the match will be set at last when you will run `/ctf_map editor`.

Author: @a-blob IIRC there is a barrier area. That's what Hobbit Power told me.

> 	- "Indestructible Red Barrier Glass" (`ctf_map:ind_glass_red`) for the build-time wall. The setting to make it disappear upon the beginning of the match will be set at last when you will run `/ctf_map editor`.

Reviewer 3: If you mean that, then yes, but it's confusing and not necessary to mention it here.

> 	- "Indestructible Red Barrier Stone" (`ctf_map:ind_stone_red`) for underground build-time wall. This will turn to Normal Stone (`default:stone`) once the match begins.

Reviewer 2: ```suggestion
	- "Indestructible Red Barrier Stone" (`ctf_map:ind_stone_red`) for underground build-time wall. This will turn to stone (`default:stone`) once the match begins.
```

> > "What about flags?," you may ask. Well, check the next section to see how to add them to your map.

Author: @a-blob , I thought it would be appropriate to add since "Team chests" are mentioned. Let's see what the other have to say about it.

> > "What about flags?," you may ask. Well, check the next section to see how to add them to your map.

Author: @a-blob should I mark this as resolved or are changes needed?

> > "What about flags?," you may ask. Well, check the next section to see how to add them to your map.

Reviewer 3: You decide, since you are the author of this pull request.

> > "What about flags?," you may ask. Well, check the next section to see how to add them to your map.

Reviewer 4: I agree with @a-blob There is no need for this sentence imo.

### 5. Fill out information about the map

Run `/ctf_map editor`.
An explanation of some of the fields is given below.

#### Map Enabled

> Whether or not the map is available for play. You will want to tick this.

Reviewer 2: ```suggestion
Whether or not the map is available for play. You will want to check this.
```

#### License
* Every map must have its own license.
* If attribution is required (for example if you modify other's map and you have to tell who is author of the original map), that has to be appended to the `license` field.
If you want to give more information, you can use the `Other info` field.
* If you don't know which license to use, [this list of CC licenses](https://creativecommons.org/use-remix/cc-licenses/) can help you.

> * We can only accept Free Software licenses, e.g. `CC BY-SA 4.0`.

Reviewer 2: ```suggestion
* We can only accept the free culture licences like `CC BY-SA 4.0`. Note that not licences in the Creative Commons family are free(as in freedom). For instance those with `ND` are not.
```

default:pick_stone,default:cobble 30,default:torch 5,ctf_ranged:pistol_loaded
```

#### Map Constants

> 1. `Map Shadow Intensity`: Sets the intensity of the shadows.

Author: In the `ctf_map editor` it was "Map Shadow Intensity".

2. `Map Gravity`: Gravitational constant of the map. (default = 1)
3. `Map Movement Speed`: Regulates the speed at which players move. (default = 1)
4. `Map Jump height`: Regulates the height of jumps. (default = 1)
5. `Map start time`: Sets time of the day the match begins at. Changing this field will instantly update the time of day in the world you are editing.
	* `0` is for midnight
	* `1000` is for 1 AM
	* `2000` is for 2 AM
	* etc.

#### Flag positions
Positions where team flags are placed. You can select the teams that you want on the map and place the flags accordingly. 

#### Zone Bounds
How far the players of a certain team may go during build time. The zone bounds should overlap at the Red Barrier wall.

> **Note:** Even standing on the boundary sends the player back. They do not have to go beyond it. Place the boundaries accordingly.

#### Chest Zones
Areas where chests can be placed. Some maps allow placing chests anywhere, while others only place them in certain locations. 

#### Barrier Area

> Area where the Indestructible "Red Barrier Glass and/or Indestructible Red Barrier Stone" wall is located. Setting this will allow the game to know where to make them disappear when the match begins.

Author: Oh is it?

> Area where the Indestructible "Red Barrier Glass and/or Indestructible Red Barrier Stone" wall is located. Setting this will allow the game to know where to make them disappear when the match begins.

Reviewer 3: I tested, it's automatically set to be the same as the map corners.

> Area where the Indestructible "Red Barrier Glass and/or Indestructible Red Barrier Stone" wall is located. Setting this will allow the game to know where to make them disappear when the match begins.

Reviewer 3: ```suggestion
Area where the Indestructible "Red Barrier Glass and/or Indestructible Red Barrier Stone" wall is located. Setting this will allow the game to know where to make them disappear when the match begins. You can leave this by default unless the game is taking a long time to remove your barriers.
```

### 6. Export

> **Note:** Only do what is instructed below if you have ***finished making the map***. 

1. Press "Finish Editing"
2. Find the exported map located in `[Mintest folder]/worlds/[Map World]/schems/[Exported Map folder]`
3. Move it to `\[Minetest folder]/games/capturetheflag/mods/ctf/ctf_map/maps`.

> ### 7. Play 

Reviewer 2: ```suggestion
### 7. Play
```

### 8. Screenshot

> If you choose to submit your map, include a screenshot of it in the exported map's folder. It should be taken with on texture packs enabled and have an aspect ratio of 3:2 (screenshot 600x400px is suggested).

Author: My bad, I was supposed to write "no" :sweat_smile:. Your edit is better. TBH, I was confused whether I did word or as no texture packs enabled or default xD.
